console/_rle_test.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+import { runLengthDecode, runLengthEncode } from "./_rle.ts";
+import { assertEquals, assertThrows } from "../assert/mod.ts";
+
+const runLengthTestCases: [number[], { d: string; r: string }, string][] = [
+  [
+    [1, 2, 3, 4, 5],
+    { d: "AQIDBAU=", r: "AQEBAQE=" },
+    "return expected value if input is normal value",
+  ],
+  [
+    [1, 1, 1, 1],
+    { d: "AQ==", r: "BA==" },
+    "return expected value if input includes an continuous value",
+  ],
+  [
+    [],
+    { d: "", r: "" },
+    "return expected value if input is empty",
+  ],
+];
```

Comment:
Are we able to turn this into an array of objects? It'd be a little easier to follow.